plugins/org.zephyrproject.ide.eclipse.core/src/org/zephyrproject/ide/eclipse/core/internal/ZephyrHelpers.java

```diff
@@ -494,4 +495,20 @@ public static String getProjectPreference(ScopedPreferenceStore pStore,
 	public static String getProjectPreference(IProject project, String key) {
 		return getProjectPreference(getProjectPreferenceStore(project), key);
 	}
+
+	/**
+	 * Parses the west arguments into a list removing unnecessary spaces
+	 *
+	 * @param args String of arguments to run west
+	 * @return List of arguments or null if there is no argument
+	 */
+	public static List<String> getArguments(String args) {
```

Comment:
Since the comment says "west arguments", could you rename the function to something like `getWestArguments()` or `parseWestArguments()`?